README.md

```diff
@@ -10,28 +10,186 @@ The features are available before an official release so that users and collabor
-- [interfaces for environments](torchrl/envs) from common libraries (OpenAI gym, deepmind control lab, etc.) and [wrappers for parallel execution](torchrl/envs/vec_env.py), as well as a new pytorch-first class of [tensor-specification class](torchrl/data/tensor_specs.py);
-- [environment transforms](torchrl/envs/transforms/transforms.py), which process and prepare the data coming out of the environments to be used by the agent;
-- various tools for distributed learning (e.g. [memory mapped tensors](torchrl/data/tensordict/memmap.py));
-- various [architectures](torchrl/modules/models/) and models (e.g. [actor-critic](torchrl/modules/tensordict_module/actors.py));
-- [exploration wrappers](torchrl/modules/tensordict_module/exploration.py) and [modules](torchrl/modules/models/exploration.py);
+- multiprocess [data collectors](torchrl/collectors/collectors.py)<sup>(2)</sup> that work synchronously or asynchronously:
+    <details>
+      <summary>Code</summary>
+    
+    ```python
+    collector = MultiaSyncDataCollector(
+        [make_env, make_env], 
+        policy=policy, 
+        devices=["cuda:0", "cuda:0"],
+        total_frames=10000,
+        frames_per_batch=50,
+        ...
+    )
+    for i, tensordict_data in enumerate(collector):
+        loss = loss_module(tensordict_data)
+        loss.backward()
+        optim.step()
+        optim.zero_grad()
+        collector.update_policy_weights_()
+    ```
+    </details>
+
+- efficient<sup>(2)</sup> and generic<sup>(1)</sup> [replay buffers](torchrl/data/replay_buffers/replay_buffers.py) that with modularized storage:
+    <details>
+      <summary>Code</summary>
+    
+    ```python
+    storage = LazyMemmapStorage(  # memory-mapped (physical) storage
+        cfg.buffer_size,
+        scratch_dir="/tmp/"
+    )
+    buffer = TensorDictPrioritizedReplayBuffer(
+        buffer_size=10000,
+        alpha=0.7,
+        beta=0.5,
+        collate_fn=lambda x: x,
+        pin_memory=device != torch.device("cpu"),
+        prefetch=10,  # multi-threaded sampling
+        storage=storage
+    )
+    ```
+    </details>
+
+- [interfaces for environments](torchrl/envs)
+from common libraries (OpenAI gym, deepmind control lab, etc.)<sup>(1)</sup> and [wrappers](torchrl/envs/vec_env.py) for parallel execution<sup>(2)</sup>, 
+as well as a new pytorch-first class of [tensor-specification class](torchrl/data/tensor_specs.py):
+    <details>
+      <summary>Code</summary>
+    
+    ```python
+    env_make = lambda: GymEnv("Pendulum-v1", from_pixels=True)
+    env_parallel = ParallelEnv(4, env_make)  # creates 4 envs in parallel
+    tensordict = env_parallel.rollout(max_steps=20)
+    assert tensordict.shape == [4, 20]  # 4 envs, 20 steps rollout
+    ```
+    </details>
+
+- cross-library [environment transforms](torchrl/envs/transforms/transforms.py)<sup>(1)</sup>, 
+executed on device and in a vectorized fashion<sup>(2)</sup>, 
```

Comment:
By vectorized fashion do you mean the SIMD acceleration in torchvision? I would just say that